cmd/exec/operations.go

```diff
@@ -439,6 +440,82 @@ func executeWorkflow(url string) (executeResponse, error) {
 
 }
 
+func executeEvent(url string) (string, error) {
+
+	// Read input data from flag file
+	inputDataOsFile, err := os.Open(localAbsPath)
+
+	if err != nil {
+		return "", err
+	}
+
+	defer inputDataOsFile.Close()
+	byteResult, _ := ioutil.ReadAll(inputDataOsFile)
```

Comment:
Don't suppress this error. File-system errors happen all of the time. The easiest way to fail here would be to have a typo in the input path, causing it to point at a file that doesn't exist.